packages/orchestrator-ui-components/src/components/WfoBadges/WfoBadge/WfoBadge.tsx

```diff
@@ -2,20 +2,23 @@ import React from 'react';
 import { FC, ReactNode } from 'react';
 
 import { EuiBadge, EuiBadgeProps, EuiText } from '@elastic/eui';
+import { TextSize } from '@elastic/eui/src/components/text/text';
 import { TextColor } from '@elastic/eui/src/components/text/text_color';
 
 export type WfoBadgeProps = EuiBadgeProps & {
     textColor: TextColor | string;
     children: ReactNode;
+    size?: TextSize;
 };
 
 export const WfoBadge: FC<WfoBadgeProps> = ({
     textColor,
     children,
+    size,
     ...restProps
 }) => (
     <EuiBadge {...restProps}>
-        <EuiText color={textColor} size="xs">
+        <EuiText color={textColor} size={size ?? 'xs'}>
```

Comment:
Aren't you reverting to a default when you create a WfoBadge already?

Reply:
^ previous comment above ^